romana-install/roles/romana/install-agent/templates/config/agent_proxy_add_policy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+body=`echo $1 | cut -d'=' -f2`
+echo Posting $body to http://{{ romana_master_ip }}:9630/
```

Comment:
It posts to localhost, not the controller's host ip.
Does it need to go to the host's `eth0` address or is localhost fine?
(If localhost is fine, it also doesn't need to be a template)